linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1132,7 +1132,19 @@ async fn test_change_voting_rights<B>(storage_builder: B) -> anyhow::Result<()>
 where
     B: StorageBuilder,
 {
-    let mut builder = TestBuilder::new(storage_builder, 4, 1).await?;
+    // To test that no fees are paid for reading or publishing committee blobs, we set the price
+    // higher than the chain balance.
```

Comment:
To be honest I can't see where this is set. I see `30_000` and `Amount::from_tokens(10)`. Is `maximum_fuel_per_block` denominated in different units than the rest of config values?

Reply:
No, it's just that we set the chain balances to 3 resp. 1 below (lines 1148 and 1149), so they couldn't pay for any of these and the blocks would fail.

Reply:
I'd extract to a variable then - like `let initial_balance = Amount::from_tokens(3)` and then set the price that is a multiple of that. It'll be clearer.